File: java/3083. Existence of a Substring in a String and Its Reverse.java
class Solution {
    public boolean isSubstringPresent(String s) {
        int n = s.length();
        for (int i = 1; i < n; i++) {
            String a = s.charAt(i) + "" + s.charAt(i-1);
            if (s.indexOf(a) != -1) return true;
        }
        return false;
    }
}

File: go/3083. Existence of a Substring in a String and Its Reverse.go
func isSubstringPresent(s string) bool {
    n := len(s)
    for i := 1; i < n; i++ {
        a := string(s[i]) + string(s[i-1])
        if strings.Contains(s, a) {
            return true
        }
    }
    return false
}

